src/main/java/com/example/gamelist/GameController.java

```diff
@@ -24,13 +27,23 @@ public  List<Game> gamelist(){
     }
 
 
+
     @PostMapping("/gamelist")
     public ResponseEntity<Response> create(@RequestBody GameForm form){
         URI url = UriComponentsBuilder.fromUriString("http://localhost:8080")
                 .path("/gamelist/id")
                 .build()
                 .toUri();
-        return ResponseEntity.created(url).body(new Response("name successfully created"));
+        return ResponseEntity.created(url).body(new Response("gamelist successfully created"));
+    }
+
+
+
+    @PatchMapping("/gamelist/{id}")
+    public ResponseEntity<UpdateResponse> UpdateGamelist(@PathVariable int id, @RequestBody GamelistService.UpdateRequest updateRequest){
```

Comment:
```suggestion
    public ResponseEntity<UpdateResponse> UpdateGamelist(@PathVariable int id, @RequestBody GamelistService.UpdateRequest updateRequest){
```
こちらの GamelistService.UpdateRequest の部分を直しましょう。

Reply:
どうなおすべきかはあえて伝えていませんので考えてほしいです。

Reply:
![スクリーンショット 2023-12-21 23 24 57](https://github.com/otyacazyu/gamelist/assets/143002834/b1bdb781-f067-4279-bd20-9ec10d247998)
572498394c9e6383d06a97d7d9cf5d7872ec07a7

GamelistService.UpdateRequestのGamelistService.を削除してみました。